bin/index.js

```diff
@@ -162,7 +158,7 @@ var debugLog = (log) => {
     
     if (options.receive) {
         const app = createDefaultApp();
-        let uploadAddress = options.ip? `${config.ssl.protocol}://${options.ip}:${options.receivePort}/receive`: `${config.ssl.protocol}://${getNetworkAddress()}:${options.receivePort}/receive`;
+        let uploadAddress = options.ip && options.receivePort ? `${config.ssl.protocol}://${options.ip}:${options.receivePort}/receive`: `${config.ssl.protocol}://${getNetworkAddress()}:${config.receivePort}/receive`;
         console.log(uploadAddress);
```

Comment:
remove this console.log and you don't need to change this line.
it’s here by mistake